source/git-util.js

```diff
@@ -38,8 +38,8 @@ exports.currentBranch = async () => {
 	return stdout;
 };
 
-exports.verifyCurrentBranchIsMaster = async () => {
-	if (await exports.currentBranch() !== 'master') {
+exports.verifyCurrentBranchIsDefault = async (defaultBranch = 'master') => {
```

Comment:
No need to define a default value here, as it's set by `meow` (see [this line](https://github.com/sindresorhus/np/pull/476/files#diff-3268f296b25109ef2e796950ea09b998R48)).